joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/testfixtures/RubyParserFixture.scala

```diff
@@ -13,7 +13,7 @@ import java.nio.charset.StandardCharsets
 import java.nio.file.{Files, Path}
 import scala.util.{Failure, Success, Using}
 
-class RubyParserFixture
+class RubyParserFixture(newMatch: Boolean = false)
```

Comment:
I would really try to avoid such a setup. It is easy to forget setting newMatch to true and then the hole test suite is useless without anybody noticing it. 
Why do we need this in the first place?